data/mods/oa_additional_buildings/json/regional_overlay.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "type": "region_overlay",
+    "id": "oa_additional_buildings_overlay",
+    "regions": [ "all" ],
+    "city": {
+      "shops": {
+        "cs_car_dealership": 1,
+        "cs_car_showroom": 1,
+        "cs_city_dump_small": 1,
+        "cs_gardening_allotment": 1,
+        "cs_internet_cafe": 1,
+        "cs_market_small": 1,
+        "cs_open_sewer": 1,
+        "cs_private_park": 1,
```

Comment:
Should any of `cs_private_park`, `cs_public_art_piece`, or `cs_public_space` be in the `parks` instead of `shops`?